comp/otelcol/otlp/components/processor/infraattributesprocessor/taggerclient.go

```diff
@@ -0,0 +1,13 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package infraattributesprocessor
+
+import "github.com/DataDog/datadog-agent/comp/core/tagger/types"
+
+type taggerClient interface {
+	Tag(entity string, cardinality types.TagCardinality) ([]string, error)
+	GlobalTags(cardinality types.TagCardinality) ([]string, error)
```

Comment:
Can you add comments for each of these functions to describe their usage? i.e.
```
// Tag does ...
Tag(entity string)...
```

Reply:
good to go thanks